terraform/api/k8s/atom.tf

```diff
@@ -100,7 +100,7 @@ resource "kubernetes_deployment" "atom" {
           image_pull_policy = "Always"
 
           resources {
-            requests {
+            requests = {
```

Comment:
@ddollar did you add these `=` in explicitly?  I don't think that's valid HCL for Terraform?

Reply:
Those = are required in the upgraded kubernetes provider

Reply:
Aaah, okay, that would be why it failed with the non-upgraded provider 😉 😆 
Will have to manage this upgrade carefully for everyone on 0.13 provisioned racks.  Get them to 0.14 first and then up to 1.0